action.yml

```diff
@@ -12,19 +12,19 @@ inputs:
   shopware-version:
     description: "Shopware Version to use"
     required: true
-    default: "v6.5.7.4"
+    default: "v6.6.8.0"
   shopware-repository:
     description: The shopware repository to checkout
     required: true
     default: shopware/shopware
   php-version:
     description: "PHP Version to use"
     required: true
-    default: "8.2"
+    default: "8.3"
   php-extensions:
     description: "PHP extensions to install"
     required: true
-    default: ""
+    default: "gd, xml, dom, curl, pdo, mysqli, mbstring, pdo_mysql, bcmath, pcov, zip"
```

Comment:
We need these extensions most of the time either way, so I'd like to propose just adding them here as a default.